src/seedu/addressbook/AddressBook.java

```diff
@@ -206,7 +206,11 @@ public static void main(String[] args) {
         showWelcomeMessage();
         processProgramArgs(args);
         loadDataFromStorage();
-        while (true) {
+        executeCommandsUntilExit();
+    }
+

```

Comment:
Can a header comment to this new method add anymore info than what the method name conveys? If so, add a header comment.
I hope you run tests after the refactoring.